src/client/components/evaluate/detail/Competitiveness.tsx

```diff
@@ -114,6 +137,38 @@ const CompetitivenessMetricDetail = ({
           )}
         </tbody>
       </Styled.table>
+      <Box>
+        {planScoreLoaded === null || planScoreLoaded === false ? (
+          <Button
+            sx={{
+              ...{
+                variant: "buttons.primary",
+                fontWeight: "light",
+                maxHeight: "34px",
+                borderBottom: "none",
+                borderBottomColor: "blue.2"
+              },
+              ...style.menuButton
+            }}
+            onClick={() => sendToPlanScore()}
```

Comment:
We should disable the button while the request is pending to avoid duplicate requests
```suggestion
            onClick={() => sendToPlanScore()}
            disabled={planScoreLoaded === false}
```